aws-lc-rs/src/aead/aes_gcm.rs

```diff
@@ -10,45 +11,89 @@ use crate::error::Unspecified;
 use aws_lc::EVP_AEAD_CTX_seal_scatter;
 use std::ptr::null;
 
+use super::{NONCE_LEN, TAG_LEN};
```

Comment:
Should we also move the items from the `use crate::aead::` line to here?

Reply:
`NONCE_LEN` is actually in `crate::nonce` and will need to remain exported via the top-level `crate::aead` module so that is a bit of a wash. We could move `TAG_LEN` here but its not really `aes_gcm` specific since the same tag length is used with chacha.